public/app/plugins/datasource/dashboard/runSharedRequest.ts

```diff
@@ -42,7 +43,12 @@ export function runSharedRequest(options: QueryRunnerOptions, query: DashboardQu
       return undefined;
     }
 
-    const listenToPanel = dashboard?.getPanelById(listenToPanelId);
+    // Source panel might be contained in a collapsed row, in which
+    // case we need to create a PanelModel
+    let listenToPanel = dashboard?.getPanelById(listenToPanelId, true);
+    if (!(listenToPanel instanceof PanelModel)) {
+      listenToPanel = new PanelModel(listenToPanel);
+    }
```

Comment:
this thing with rows and collapsed rows not containing PanelModels feels so messy. 

I wonder if it's better to say this scenario is not supported (Sharing a panel in a collapsed row), and figure out how to support it in scenes

Reply:
I agree it's not ideal, but as a stopgap until scenes is fully ready maybe it's acceptable?

Reply:
hi team, any idea if this will pass on or it remains as a feature request?

Reply:
@kaydelaney @dprokop 

Just a bit afraid of this solution that creates this floating / disconnected PanelModel instance. If you expand the row where this panel is inside a new model will be created again and a new query issued. 

So afraid we will quickly get a new bug report saying the query is not shared (re-issued) when the row is expanded